x/ecocredit/params.go

```diff
@@ -9,10 +9,10 @@ import (
 
 var (
 	// TODO: Decide a sensible default value
-	DefaultCreditClassFeeTokens  = sdk.NewInt(10000)
-	KeyCreditClassFee            = []byte("CreditClassFee")
-	KeyAllowlistedCreditCreators = []byte("AllowlistCreditCreators")
-	KeyAllowlistEnabled          = []byte("AllowlistEnabled")
+	DefaultCreditClassFeeTokens   = sdk.NewInt(10000)
+	KeyCreditClassFee             = []byte("CreditClassFee")
+	KeyAllowlistedCreditDesigners = []byte("AllowlistCreditDesigners")
```

Comment:
why not using `KeyAllowedClassDesigners ` as in the params field `AllowedClassDesigners`?